Src/Couchbase.Linq/BucketContext.cs

```diff
@@ -22,7 +22,7 @@ namespace Couchbase.Linq
     /// </summary>
     public class BucketContext : IBucketContext, IChangeTrackableContext
     {
-        private readonly ConcurrentDictionary<Type, PropertyInfo>_cachedKeyProperties = new ConcurrentDictionary<Type, PropertyInfo>();
+        private static readonly ConcurrentDictionary<Type, PropertyInfo> s_cachedKeyProperties = new ConcurrentDictionary<Type, PropertyInfo>();
```

Comment:
Typically we follow ReSharper's code styling which would recommend a capitalised static variable name; 
 eg *CachedKeyProperties*